Tests/MongoSwiftTests/MongoCollection+IndexTests.swift

```diff
@@ -78,13 +79,22 @@ final class MongoCollection_IndexTests: MongoSwiftTestCase {
             return
         }
 
+        let sv = try _client?.serverVersion() ?? ServerVersion("8000")
+        guard try sv < ServerVersion("4.3.0") else {
+            print("Skipping test, fails with latest server version because of missing ns field")
+            return
+        }
+
+        let nsString = MongoSwiftTestCase.isMacOS ? "test.MongoCollection_IndexTests_testIndexOptions" :
+                                                    "test.MongoCollection_IndexTests.testIndexOptions"
+
```

Comment:
The `ns` field seems to be different based on whether it is running on MacOS ("_") or Ubuntu ("."). This was the only way I could get the evergreen tests to pass, but please let me know if there is a better way to do this.

Reply:
So, upon further reflection, I've realized that setting the `ns` field on `IndexOptions` doesn't actually make a lot of sense. When calling `listIndexes` on a particular `MongoCollection`, you know that all indexes returned from that query will have the same namespace (that of the collection you called the method on). Likewise for `createIndex`, we wouldn't want to allow the user to create an index on a different namespace from the collection they were invoking the command on. Also, as proved by the tests that were failing on "latest", the server isn't going to even include `ns` in future responses from the server. For all those reasons, I think it's okay to just omit the field altogether. Worst case, we can add it back later for spec compliance. 

Sorry for sending the wrong signals on what to do on this one--I hadn't read the spec in its entirety and made some incorrect assumptions.